netbox/ipam/filtersets.py

```diff
@@ -1089,6 +1091,20 @@ def filter_vminterface_id(self, queryset, name, value):
         )
 
 
+class VLANTranslationPolicyFilterSet(NetBoxModelFilterSet):
```

Comment:
Both filtersets need a `search()` method to allow searching on their name & description.